spring-boot-email-core/src/main/java/it/ozimov/springboot/templating/mail/utils/MimeMessageHelperExt.java

```diff
@@ -0,0 +1,49 @@
+package it.ozimov.springboot.templating.mail.utils;
+
+import org.springframework.mail.javamail.MimeMessageHelper;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * Created by ssanchezc on 23/12/2016.
+ */
+public class MimeMessageHelperExt extends MimeMessageHelper {
```

Comment:
hi @robertotru , i can´t use MimeMessageHelper  because this class is in package spring org.springframework.mail.javamail,that's why i extend this classs